app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -70,7 +81,61 @@ public void onClick(View v) {
         });
         return rootview;
     }
+    public List<EnhancementOptionsEntity> getEnhancementOptions() {
+        mTextEnhancementOptionsEntityArrayList.clear();
 
+        mTextEnhancementOptionsEntityArrayList.add(
+                new EnhancementOptionsEntity(getResources().getDrawable(R.drawable.ic_font_black_24dp),
+                        getResources().getString(R.string.edit_font_size)));
+        return mTextEnhancementOptionsEntityArrayList;
+    }
+    private void showEnhancementOptions() {
+        GridLayoutManager mGridLayoutManager = new GridLayoutManager(getActivity(), 2);
+        mTextEnhancementOptionsRecycleView.setLayoutManager(mGridLayoutManager);
+        mTextEnhancementOptionsAdapter = new EnhancementOptionsAdapter(this, getEnhancementOptions());
+        mTextEnhancementOptionsRecycleView.setAdapter(mTextEnhancementOptionsAdapter);
+    }
+    @Override
+    public void onItemClick(int position) {
+        switch (position) {
+            case 0:
+                editFontSize();
+                break;
+        }
+    }
+    private void editFontSize() {
+        new MaterialDialog.Builder(mActivity)
+                .title(R.string.edit_font_size)
+                .content(R.string.enter_font_size)
+                .input(getString(R.string.example_font), null, new MaterialDialog.InputCallback() {
+                    @Override
+                    public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
+                        try {
+                            mFontSize = Integer.parseInt(String.valueOf(input));
+                            if (mFontSize > 1000 || mFontSize < 0) {
+                                Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
+                                        R.string.invalid_quality,
+                                        Snackbar.LENGTH_LONG).show();
+                            } else {
+                                showFontSize();
+                                Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
+                                        R.string.font_size_changed,
+                                        Snackbar.LENGTH_LONG).show();
+                            }
+                        } catch (NumberFormatException e) {
+                            Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
+                                    R.string.invalid_quality,
```

Comment:
change the name of string `invalid_quality` to `invalid_entry` (and make changes in other files too)